src/commands/mod.rs

```diff
@@ -34,9 +33,10 @@ pub enum Command {
 }
 
 impl Command {
-    pub fn execute<W: Write>(self, writer: &mut W) -> Result<(), Error> {
+    pub fn execute(self, env: impl Env) -> Result<(), Error> {
         match self {
-            Self::Nsec3Hash(nsec3hash) => nsec3hash.execute(writer),
+            Self::Nsec3Hash(nsec3hash) => nsec3hash.execute(env),
+            Self::Key2ds(key2ds) => key2ds.execute(env),
```

Comment:
Why does this PR for NSEC3 hashing add a mapping for `key2ds`?

Reply:
Looks like the merge didn't get recognized by github

Reply:
Seems like this is only shown as a change if showing the recent changes/commits (e.g. since last review) and not when viewing all commits.